test/test.js

```diff
@@ -87,6 +87,71 @@ describe('http-signature', () => {
         `host: example.com:18443\ndate: ${date}\n(request-target): get /1/2/3`);
       done();
     });
+    it('properly encodes `(created)` with a timestamp', done => {
```

Comment:
if tests don't have any async, we don't need `done` right?  And if they are async we should be using async/await.

Reply:
good catch here and yeah createSignatureString is not async.
looks like I copied someone who in turn copied someone doing async testing.

Reply:
addessed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/c3db9340863113950472faa881437a6dcc154eaa